### PR TITLE
Fix missing config keys

### PR DESCRIPTION
**Pull request by jkanem**
- [x] I have added the tests to cover my changes.
- [x] I have updated the documentation, VERSION, and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

File: tests/functional_tests/functional_test_files/covalent/covalent.conf [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[sdk]
```
Comment (FyzHsn) [inline tests/functional_tests/functional_test_files/covalent/covalent.conf:1]: Just curious, what was the reasoning behind choosing the functional tests directory as the home for these tests?

**Review by FyzHsn — CHANGES_REQUESTED** [review]

**Comment by FyzHsn** [comment]
@jkanem Just had a question that requires some clarification and a minor change.

**Comment by FyzHsn** [comment]
@jkanem I think tests/covalent_tests/shared_files/config_tests.py would be a good place to move the tests to.

File: tests/functional_tests/config_test.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+
+import os
+from unittest.mock import patch
+
+from covalent._shared_files.config import _ConfigManager
+
+config_dir = os.path.join(os.path.dirname(__file__), "functional_test_files")
```
[the remark this answers is not in the document]
Comment (jkanem) [reply tests/functional_tests/config_test.py]: Done

File: tests/functional_tests/functional_test_files/covalent/covalent.conf (hunk 1 above)
Comment (jkanem) [reply tests/functional_tests/functional_test_files/covalent/covalent.conf]: I moved it (and the sample config file) to covalent_tests/shared_files/

**Review by FyzHsn — APPROVED** [review]

**Pull request merged**